pipeline/test/event.test.ts

```diff
@@ -11,8 +11,8 @@ describe("time window events", () => {
       const result = toBoundedWindow(event);
 
       expect(result.end).toBeDefined();
-      expect(result.end?.getTime()! - result.start?.getTime()!).toBe(
-        60 * 60 * 1000
+      expect(result.end!.getTime() - result.start!.getTime()).toBe(
+        60 * 60 * 1000,
```

Comment:
My favourite telling off:
> Optional chain expressions can return undefined by design - using a non-null assertion is unsafe and wrong.

Here I assumed we knew end and start would always have a value

Reply:
> is unsafe and wrong.

There are a few times in my life when I wish I'd had this warning 😅

Reply:
Just to clarify these are not Prismic events, but the event that trigger the lambda. 
This is testing that in the case where we have a `duration` and a `start` in the aforementioned event then the so-called "boundedWindow" does have an end and the time between start and end is as expected

Reply:
Do you specify because you think the test could be clearer? (as in should we amend the description?)